### ISSUE TITLE
Error using vl_nnconv DEROUTPUT dimensions are incompatible with X and FILTERS.

### ISSUE THREAD

**Issue by kbenahmed89** [vlfeat/matconvnet#428]
I'm trying to tune a pretrained VGG-F model, I want to retrain it on my small dataset which is MRI images. 
this is my code:

load('imdb.mat') ;
imdb.images.data = single (imdb.images.data );
net = load('imagenet-vgg-f.mat') ;
[net,info] = cnn_train(net, imdb, @getBatch, trainOpts) ;

function [im, labels] = getBatch(imdb, batch)
% --------------------------------------------------------------------
im = imdb.images.data(:,:,:,batch) ;
labels = imdb.images.label(1,batch) ;

and the code I used to replicate my 22 grayscale images on 3 channels:
for i=1:22

rgb(: , : , : , i ) = repmat(grayimages( : , : , i ), [1 1 3]);

end

I'm getting this error ,can you please help me. I'm stuck.

**Comment by hannanow** [comment]
I think you have to add some loss layer as the last layer to your network to be able to train it. So my guess is that it says the size of derivatives backpropagated are incompatible which comes from the fact they simply don't exist as there is no 'loss' layer to 'produce them'.
